html/changelogs/Kurfursten1911.yml

```diff
@@ -1,2 +1,2 @@
 author: Kurfursten
-changes: []
+changes: [Adds special sounds for prayers and faxes to admins -- hopefully you'll get noticed now! Also faxes cooldown faster.]

```

Comment:
I do not believe this is going to work.
Try making a new changelog file for this PR, and follow the example.yml formatting.